### ISSUE TITLE
how do i get video chapters embed into the video

### ISSUE THREAD

**Issue by nagaisa** [ytdl-org/youtube-dl#28926]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

so i am downloading a long video and its slipt into many chapters i was wondering how i could get youtube-dl to do this
i have tried looking many places but right now i can't view them because of it being blocked where i am (the sites) 

in short, is there a way to download the chapter info and embed it into the file?

**Comment by dstftw** [comment]
`--add-metadata`.

**Comment by nagaisa** [comment]
I think I did that but did not work
